dev-python/pyilmbase/pyilmbase-2.5.2.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
what about py39?

Reply:
Haven't tested it yet, as the interpreter is still `_beta`. Going to test, whether it's working.

Reply:
Boost doesn't have support for py39 yet.